cypress/support/step-definitions/radio-button-steps.js

```diff
@@ -109,3 +116,29 @@ Then('{string} RadioButton label width is set to {string}', (radioButtonName, wi
 When('I set RadioButton {word} {word} slider to {int}', (propertyName, text, width) => {
   setSlidebar(labelWidthSliderByName(propertyName, text), width);
 });
+
+Then('Then RadioButton are inline', () => {
+  radioButtonGroup().should('have.css', 'display', 'flex');
+  radioButtonComponentByPosition(FIRST_RADIOBUTTON).should('have.css', 'margin-left', '0px');
+  radioButtonComponentByPosition(SECOND_RADIOBUTTON).should('have.css', 'margin-left', '32px');
+  radioButtonComponentByPosition(THIRD_RADIOBUTTON).should('have.css', 'margin-left', '32px');
+});
+
+Then('Then RadioButton are not inline', () => {
+  radioButtonGroup().should('have.css', 'display', 'block');
+  radioButtonComponentByPosition(FIRST_RADIOBUTTON).should('have.css', 'margin-left', '0px');
+  radioButtonComponentByPosition(SECOND_RADIOBUTTON).should('have.css', 'margin-left', '0px');
+  radioButtonComponentByPosition(THIRD_RADIOBUTTON).should('have.css', 'margin-left', '0px');
+});
+
+Then('Then legend is inline with RadioButton', () => {
+  radioButtonFieldset().should('have.css', 'display', 'flex');
+  radioButtonLegend().should('have.css', 'margin-right', '32px')
+    .and('have.css', 'height', '34px');
+});
+
+Then('Then legend is not inline with RadioButton', () => {
```

Comment:
As above - We don't need `Then` here, because BDD step itself is called `Then`

```suggestion
Then('legend is not inline with RadioButton', () => {
```